core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -586,6 +586,7 @@ class AutoTuner(
     appendRecommendation("spark.rapids.shuffle.multiThreaded.writer.threads", numExecutorCores)
     appendRecommendation("spark.rapids.sql.multiThreadedRead.numThreads",
       Math.max(20, numExecutorCores))
+    appendRecommendation("spark.rapids.sql.batchSizeBytes", 2147483647)
```

Comment:
1. Can we have `2147483647` as a constant along with a comment that explains why do we have that value as the default?  It will be easier to maintain the code when we have those hardcoded defined as constants, or pulled from default settings. It can also be used by the unit tests.
2. Is it part of the reqirements to recommend `batchSizeBytes` regardless the value of `spark.sql.adaptive.enabled`?

Reply:
Thank you!
1. Fixed.
2. Yes, we intend to recommend `batchSizeBytes` regardless the value of `spark.sql.adaptive.enabled`.